ClientStateListenerTest Failures

```
java.lang.AssertionError: null
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at com.hazelcast.client.util.ClientStateListenerTest.testClientConnectedWithTimeout(ClientStateListenerTest.java:239)
```

https://hazelcast-l337.ci.cloudbees.com/job/new-lab-fast-pr/9708/testReport/junit/com.hazelcast.client.util/ClientStateListenerTest/testClientConnectedWithTimeout/
https://hazelcast-l337.ci.cloudbees.com/job/new-lab-fast-pr/9709/testReport/com.hazelcast.client.util/ClientStateListenerTest/testClientConnectedWithTimeout/
https://hazelcast-l337.ci.cloudbees.com/job/new-lab-fast-pr/9711/testReport/com.hazelcast.client.util/ClientStateListenerTest/testClientConnectedWithTimeout/

--- a/hazelcast-client/src/test/java/com/hazelcast/client/util/ClientStateListenerTest.java
+++ b/hazelcast-client/src/test/java/com/hazelcast/client/util/ClientStateListenerTest.java
@@ -236,7 +236,7 @@ public class ClientStateListenerTest extends ClientTestSupport{
         hazelcastFactory.newHazelcastInstance();
 
         hazelcastFactory.newHazelcastClient(clientConfig);
-        assertTrue(listener.awaitConnected(1, MILLISECONDS));
+        assertTrue(listener.awaitConnected(30, SECONDS));
 
         assertFalse(listener.awaitDisconnected(1, MILLISECONDS));
 

